eisbuk-admin/src/pages/debug/index.tsx

```diff
@@ -61,7 +61,7 @@ const DebugPage: React.FC = () => {
       </Box>
       <Box my={4} color="secondary.main">
         <Button
-          onClick={invokeFunction(CloudFunction.CreateTestSlots)}
+          onClick={() => invokeFunction(CloudFunction.CreateTestSlots)()}
```

Comment:
I don't get the reason for this change.

`invokeFunction` might be poorly named: it makes you think that when you invoke `invokeFunction` a cloud function will be invoked, as a side effect. But instead a function is returned that can be used to invoke the cloud function.
This naming can be improved, but in general code like

```javascript
const something = () => somefunction()();
```

should be written like this instead:
```javascript
const something = somefunction();
```

Reply:
I don't have an idea why, either. However, when invoking the "old" way, we get the console.error "max call stack size increased" and function (cloud function) doesn't get called. I believe it might have something to do with React's rendering cycle and maybe `useCallback` could fix this issue, but it's more straighforward this way...

Reply:
Try the existing `master` and I believe you will be able to reproduce (max call stack size)

Reply:
Thanks for the explanation and fix. I think I noticed the problem but didn't get to fix it.
I'm glad you took care of it!